src/Components/Toast.js

```diff
@@ -1,15 +1,15 @@
 import { useEffect } from "react";
-import { useWish } from "../Context";
+import { useData } from "../Context";
 export const Toast = () => {
-  const { toast, setToast, toastMessage } = useWish();
+  const { toast, setToast, toastMessage } = useData();
   useEffect(() => {
     const toastTimeout = setTimeout(() => {
       setToast(false);
     }, 1000);
     return () => {
       clearTimeout(toastTimeout);
     };
-  }, [toast,setToast]);
+  }, [toast, setToast]);
   return (
     <div
       style={{
```

Comment:
try to move wherever inline style is applied, to a css file, then import and use